testing-and-tdd/jasmine-testing.md

```diff
@@ -25,173 +25,362 @@ Jasmine is a Behavior Driven Development testing framework for JavaScript. It do
 
 *Participants will be able to:*
 
-- create a testing structure
-- create assertion functions
+- Create a testing structure with Jasmine
+- Create assertion functions
 - Generate, display and watch tests
 
 ### Specific Things to Learn
 
 - [Jasmine](https://jasmine.github.io/index.html)
   - install Jasmine
-  - create tests using `toBe`, `beforeEach`, `afterEach`
-- [Mocha](https://mochajs.org/)
-  - install Mocha
-  - create tests using assert
-- [Chai](https://www.chaijs.com/)
-  - install Chai
-  - require library for assertions
+  - create tests using `expect` & matchers like `toBe`, `toContain`, and `toBeDefined`
 
 ### Materials
 
-- [Jasmine Testing Slideshow](https://docs.google.com/presentation/d/1V9WEprK9j61V0S1lYtcJOXNTDMpziVMyUKHuMhpP1W0/edit?usp=sharing)
+- [Jasmine and Mocha/Chai Testing Slideshow](https://docs.google.com/presentation/d/1jMkVbUkZS40z7kAoC1mPIt8RwfF8-GRL8oeee25UzYw)
+    - Note: you can skip over the Mocha / Chai sections of the slide show for now. We'll go over that in the next lesson!
 
 ### Lesson
 
 #### Jasmine Test
-Jasmine is a behavior-driven development (BDD) for testing JavaScript code. Jasmine has no external dependencies and does not require a DOM. 
+Jasmine is a behavior-driven development (BDD) framework for testing JavaScript code. (Note: BDD is a specific style of test-driven development that tests the behavior of the code from the user's perspective.) Jasmine has no external dependencies and does not require a DOM, which means that it's good for getting tests up and running quickly.
```

Comment:
Would you mind providing a link on BDD in case they'd like to learn more?

Also, BDD and TDD aren't mutually exclusive?

Reply:
👍 Added a link to learn more!

> Also, BDD and TDD aren't mutually exclusive?
From what I've read, TDD is more of a process for writing tests (start with test, then write code) whereas BDD is more of a "how/what to test". Added a little wording tweak that hopefully clarifies, but let me know if I can clarify further!